src/Personal/reducer.js

```diff
@@ -0,0 +1,67 @@
+import { actionType } from './actionCreator';
+
+const initialBackup = {
+  response: 'Click button to backup data',
+};
+
+const initialUser = {
+  username: '',
+  email: '',
+  accessToken: '0xa143981f3ec758296a1575146eab03ef047b7e40',
+};
+
+const backupState = (state = initialBackup, action) => {
+  switch (action.type) {
+    case actionType.BACKUP_START:
+      return {
+        ...state,
+        response: 'Backing up',
```

Comment:
do not communicate the current state as a readable string. The reducer stores values for the code, not for the user. 
Backing up means it is in progress, so create a new key like `isInProgress: true/false`. You will decide what to display within the component, the UI (Presenter) layer. 
```jsx
{
  isInProgress ? <Text>Back up in progress</Text> : <BackupButton />
}
```

Reply:
Done